perf_sim/config.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ * config.cpp - implementation of Config class
+ * @author Pavel Kryukov pavel.kryukov@phystech.edu
+ * Copyright 2017 MIPT-MIPS
+ */
+
+/* Simulator modules */
+#include "config.h"
+
+/* Generic C++ */
+#include <iostream>
+
+/* boost - program option parsing */
+#include <boost/program_options.hpp>
+
+using namespace std;
+
+/* constructor and destructor */
+Config::Config() :
+    /* by default, there is no limit of simulation and silent mode is on */
+    num_steps( -1),
```

Comment:
Boost allows to set default values to options, please use that feature to keep everything in one place.